docs/releases/saros-e_16.0.0.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Saros for Eclipse 16.0.0 Release Notes
+---
+
+:tada: We are happy to announce **Saros for Eclipse 16.0.0** :tada:
+
+The new Saros for Eclipse 16.0.0 contains multiple [bug fixes](#bug-fixes)
+and [improvements](#improvements).
+
+## Installation
+Saros for Eclipse 16.0.0 requires
+ - `JDK 8` or newer
+ - Eclipse 4.6 (Neon) or newer
+
+## Compatibility
+Saros for Eclipse 16.0.0 is not compatible with other Saros versions.
+
+## Feedback
+If you find any issue with Saros, please report it in our [issue tracker](https://github.com/saros-project/saros/issues).
+In case you are not sure whether you found a bug or simply want to talk about Saros, ask us at [Gitter](https://gitter.im/saros-project/saros/user).
+
+## Changes
+
+Beside multiple internal reworks, [bug fixes](#bug-fixes) and [improvements](#improvements) we also
+[removed our Whiteboard](https://www.saros-project.org/contribute/deprecated/whiteboard.html).
+It was to brittle and does not fit into our plan to support further Editors/IDEs in the future.
+
+### Improvements
+* The `.git` directory is now ignored by default during a session ([#922](https://github.com/saros-project/saros/issues/922)).
+* Add Contacts Context Menu Item is enabled for Offline Contacts ([#19](https://github.com/saros-project/saros/issues/19))
+* Choose only one "XMPP ID" vs "Jabber ID" ([#22](https://github.com/saros-project/saros/issues/22))
+* Reusing existing project requires cumbersome manual selection ([#66](https://github.com/saros-project/saros/issues/66))
```

Comment:
This text seems to describe the previous behavior that was "fixed"/improved. This is kind of confusing as it is not directly apparent that these issues have been resolved.

I would suggest rewording it to make it more clear either that this was a problem that has been fixed or what the new behavior is.

The same also applies to the next section `Bug Fixes`. I would suggest changing the wording to show that the issues were resolved (e.g. by using past tense). Alternatively, you could rename the section to `Fixed Bugs`.

Reply:
I will change there wording and will rename it to fixed bugs :+1: